two_factor/views/core.py

```diff
@@ -48,7 +49,7 @@
 
 @class_view_decorator(sensitive_post_parameters())
 @class_view_decorator(never_cache)
-class LoginView(IdempotentSessionWizardView):
+class LoginView(TemplateView):
```

Comment:
Because I wanted to make sure the security issue caused by the inheritance was fixed and would not come back by accident.